src/utils/bit_hacks.hpp

```diff
@@ -90,18 +90,18 @@ inline int NumberOfBinaryTrailingZeros(std::uint64_t val) {
 
 inline int MaximumPowerOf2Divisor(int in) { return in & (~(in - 1)); }
 
-inline uint IntegerLog2Ceil(uint in) {
-  uint log2 = 0;
-  uint in_temp = in;
+inline unsigned int IntegerLog2Ceil(unsigned int in) {
+  unsigned int log2 = 0;
+  unsigned int in_temp = in;
```

Comment:
This is unrelated but my compiler on my laptop yelled at me for `uint` which is apparently not technically part of the standard.